c-sharp/README.md

```diff
@@ -0,0 +1,74 @@
+﻿MongoDB Charts Embedding Example - Verified Signature Auth in C#
+================================================================
+
+Background
+----------
+MongoDB Charts allows you to create visualizations of your MongoDB data using a simple web interface.
+You can view the visualizations within the Charts UI, or you can use the Embedding feature to render
+the charts in an external web application. When you embed a chart, you can choose whether you want
+it to be Unauthenticated (meaning anyone who has the embed code can view the chart), or to use
+"Verified Signature" authentication (which checks the integrity of a signature in the URL before the
+chart will render). The Verified Signature option requires a few more steps to implement but it is
+more secure -- your app will generate the signature on the server side after first authenticating
+and authorizing the user, and the URL will stop working after a specified time period. 
```

Comment:
actually, rather than repeating this in every language, should we move the general introduction to the top level into the README.md file?

Reply:
While it's more work for us to maintain, I thought it was better to keep it in each sample. A user who is interested in a specific language probably wouldn't bother even looking at the other folders, including the top level. This also means we can have a single README file for each sample that has everything they need to know.